Sources/Knit/Container.swift

```diff
@@ -11,24 +11,11 @@ import Swinject
 
  The Knit.Container also performs the function of a weak wrapper of the `Swinject.Container`.
  */
-public class Container<TargetResolver>: Knit.Resolver {
+public class Container<TargetResolver: Resolver> {
```

Comment:
Seems like this should still specify `Knit.Resolver` as the required conformance constraint?
```suggestion
public class Container<TargetResolver: Knit.Resolver> {
```